Jenkinsfile_promote_salt_packages

```diff
@@ -9,49 +9,83 @@ properties([
+                                sh "git push origin openSUSE/MU/${mu_version}"
+                            }
+                            echo "Successfully created and pushed 'openSUSE/MU/${mu_version}' branch to 'openSUSE/salt' repository"
+                            sh "git clone --branch release/${salt_version} --depth 1 https://github.com/openSUSE/salt-packaging"
+                            dir('/tmp/salt-promote-pipeline-env/salt-packaging') {
+                                sh "git checkout -b MU/${mu_version}"
+                                sh "git push origin MU/${mu_version}"
+                            }
+                            echo "Successfully created and pushed 'MU/${mu_version}' branch to 'openSUSE/salt-packaging' repository"
+                        }
+                        else {
+                            if (params.recreate_salt_mu_branches) {
+                                // Forcing recreation of MU branches
+                                echo "The MU branches already exist, but pipeline was set to recreate them"
+                                sh "git clone --branch openSUSE/release/${salt_version} --depth 1 https://github.com/openSUSE/salt"
+                                dir('/tmp/salt-promote-pipeline-env/salt') {
+                                    sh "git checkout -b openSUSE/MU/${mu_version}"
+                                    sh "git push -f origin openSUSE/MU/${mu_version}"
+                                }
+                                echo "Successfully recreated and pushed 'openSUSE/MU/${mu_version}' branch to 'openSUSE/salt' repository"
+                                sh "git clone --branch release/${salt_version} --depth 1 https://github.com/openSUSE/salt-packaging"
+                                dir('/tmp/salt-promote-pipeline-env/salt-packaging') {
+                                    sh "git checkout -b MU/${mu_version}"
+                                    sh "git push -f origin MU/${mu_version}"
+                                }
+                                echo "Successfully recreated and pushed 'MU/${mu_version}' branch to 'openSUSE/salt-packaging' repository"
+                            }
+                            else {
+                                error("The MU branches were already created. Default promotion is not possible. Exiting.")
+                            }
+                        }
+                        sh "rm salt -rf && rm salt-packaging -rf"
+                    }
 
-                echo "Check the source tarball is properly named to salt_${salt_version}.orig.tar.gz in 'products:testing:debian'"
-                sh "curl https://build.opensuse.org/package/view_file/systemsmanagement:saltstack:products:testing:debian/salt/_service?expand=0 | grep salt_${salt_version}.orig.tar.g"
-            }
-        }
+                    echo "Check that 'products:testing' and 'products:testing:debian' are not set to MU branches"
```

Comment:
This has to do with how we were doing for building the DEB packages, where pipeline was by then adjusting the branches configured at `_service`  files. Now, this is done differently, so the branch configured in `_service` file are not expected to change during the execution of the pipeline.

In `products:testing` and `products:testing:debian`, we have `_service` configured to point to our devel branch: `openSUSE/release/3006.0` (not really to an MU branch). This way, any time we push something new to `products:testing`, the services would run at `products:testing:debian` updating the tarball that is used to build DEB package.

The pipeline then expects that this is the case, so when `_service` is pointing to an MU branch, then this means something unexpected had happened and we don't really want to run the promotion at this point.

If the `_service` is NOT pointing to an MU branch, then pipeline assumes we are in a good shape, and will run the promotion.

We can keep this checks, which are harmless at the moment, and will makes us to notice in case the branches has changed.